uctMaths/competition/compadmin.py

```diff
@@ -685,6 +685,9 @@ def assign_student_awards():
                         student.save()
                     else:
                         break
+    
+    student_list = SchoolStudent.objects.all()
```

Comment:
What does this line do?

Reply:
I think that was left over from the earliest tests I did - I thought I removed it before, but it should be gone now